src/managers/poetry/poetryUtils.ts

```diff
@@ -330,8 +329,8 @@ function nativeToPythonEnv(
         tooltip: info.prefix,
         execInfo: {
             run: { executable: info.executable },
-            shellActivation,
-            shellDeactivation,
+            shellActivation: pythonInfo.execInfo.shellActivation,
```

Comment:
Something about this is weird. The refactoring should have made it so that you pass the native env info get back just the activation. Here were are creating env info twice.